qrisk/tests/test_stats.py

```diff
@@ -79,49 +137,140 @@ def test_annual_volatility(self, returns, period, expected):
         )
 
     @parameterized.expand([
-        (simple_rets, 1.2333396776895436),
+        (s_2.pct_change().dropna(), -2.3992211554712197)
+    ])
+    def test_calmar(self, returns, expected):
+        self.assertAlmostEqual(
+            stats.calmar_ratio(
+                returns),
+            expected,
+            DECIMAL_PLACES)
+
+    @parameterized.expand([
+        (s_1, 0.0, 2.0)
+    ])
+    def test_omega(self, returns, minimum_acceptable_return, expected):
+        self.assertEqual(
+            stats.omega_ratio(
+                returns,
+                minimum_acceptable_return=minimum_acceptable_return),
+            expected)
+
+    @parameterized.expand([
+        (simple_rets, 1.2321057207245873),
         (np.zeros(10), np.nan),
         ([0.1, 0.2, 0.3], np.nan)
     ])
-    def test_sharpe(self, returns, expected):
+    def test_sharpe_ratio(self, returns, expected):
         assert_almost_equal(
             stats.sharpe_ratio(
                 np.asarray(returns)),
             expected, DECIMAL_PLACES)
 
     @parameterized.expand([
-        (pd.Series(px_list_2,
-                   index=dt_2).pct_change().dropna(), -2.3992211554712197)
+        (simple_rets, 0.0, stats.DAILY, 0.0),
+        (simple_week_rets, 0.0, stats.WEEKLY, 0.0),
+        (simple_month_rets, 0.0, stats.MONTHLY, 0.0),
+        (simple_rets, 0.1, stats.DAILY, 1.5826812692390093),
+        (simple_week_rets, 0.1, stats.WEEKLY, 0.718943669559723),
+        (simple_month_rets, 0.1, stats.MONTHLY, 0.3453693674893592),
+        (df_simple, 0.0, stats.DAILY,
```

Comment:
The first three test when downside risk should be 0 at different time periods. The fourth through sixth test the same inputs but with a higher minimum acceptable return, again for different time periods. The last three test when the input is a dataframe and I believe the values for 'one' and 'two are a list of random  numbers with a mean of 0 and 0.15.